src/test/compile-fail/array_const_index-1.rs

```diff
@@ -7,6 +7,7 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+#![feature(const_indexing)]
```

Comment:
These two tests started failing because the `const_indexing` error was being reported instead of the index out of bounds error.